src/test/checkValidTargetFilters.spec.ts

```diff
@@ -0,0 +1,72 @@
+import * as chai from "chai";
+chai.use(require("chai-as-promised"));
+const expect = chai.expect;
```

Comment:
We hook in `chai-as-promised` in our bootstrap file in tests, so you can reduce this to:

```suggestion
import { expect } from "chai";
```